.buildlibrary

```diff
@@ -1,6 +1,7 @@
-ValidationKey: '23354100'
+ValidationKey: '23718520'
 AcceptedWarnings:
 - 'Warning: package ''.*'' was built under R version'
 - 'Warning: namespace ''.*'' is not available and has been replaced'
 - ‘qpdf’ is needed for checks on size reduction of PDFs
 AcceptedNotes: unable to verify current time
+AutocreateReadme: yes
```

Comment:
If possible, can you please also add github actions?

Reply:
looks like it's worked: https://github.com/giannou/mip/actions/runs/581231478